compiler-rt/lib/sanitizer_common/sanitizer_freebsd.h

```diff
@@ -1,137 +0,0 @@
-//===-- sanitizer_freebsd.h -------------------------------------*- C++ -*-===//
```

Comment:
why is this file no longer needed

Reply:
https://github.com/llvm/llvm-project/commit/1f386f0f364183d85d636ff7be160712fc4bb5b8

The header was added as part of a workaround for 9.x and after that is removed it is basically an empty header. Just...
```
#if SANITIZER_FREEBSD

#endif
```